01 - Getting Started.ipynb

```diff
@@ -24,13 +24,13 @@
     "### `conda` / `mamba`\n",
```

Comment:
<p>This import does not match what's on the readme, where you have</p><p> </p><p>#### conda / mamba</p><p> </p><p><code></code><code>sh</p><p>mamba create -n ibis-tutorial python ibis-duckdb=7 python-duckdb=0.8.1 jupyterlab altair plotnine</p><p></code><code></code></p><p> </p><p>#### pip</p><p> </p><p><code></code><code>sh</p><p>python -m pip install 'ibis-framework[duckdb]==7' duckdb==0.8.1 jupyterlab altair&gt;=5.0.1 plotnine</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>